src/components/PriceChange.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React, { Component } from "react";
+import LText from "./LText";
+import colors from "../colors";
+
+export default class PriceChange extends Component<{
+  before: number,
+  after: number,
+  color?: "western" | "eastern",
+  style?: {}
+}> {
+  render() {
+    const { before, after, color } = this.props;
+    const style: { color?: * } = {};
+
+    if (!before) {
+      return <LText />;
+    }
+
+    const percent: number = after / before * 100 - 100;
+
+    if (color === "western") {
+      style.color = percent >= 0 ? colors.green : colors.red;
+    } else if (color === "eastern") {
+      style.color = percent >= 0 ? colors.red : colors.blue;
+    }
+
+    return (
+      <LText style={[this.props.style, style]}>
+        {`${(percent >= 0 ? "+" : "") + percent.toFixed(2)} %`}
```

Comment:
according to desktop wireframes, we want a space between the sign and the number. so you might have to `?"+ ":"- "` and to `Math.abs(percent)..`

Reply:
I followed the poc mobile screen design, my bad.